public/app/src/stroke.js

```diff
@@ -0,0 +1,19 @@
+function Stroke() {
+  this.size = 10;
+  this.currentColour = '#000';
+  this.red = '#ff0000';
```

Comment:
Perhaps we don't need these colour properties (red, etc.) now. Presumably, the colour will always be set via changeColour and some value that comes from the DOM.